labels.rb

```diff
@@ -4,17 +4,41 @@
 require_relative 'octokit_utils'
 require_relative 'options'
 
+def primary_remote
+  ['upstream', 'origin'].map do |name|
+    get_remote(name)
+  end.compact.first
+end
+
+def get_remote(name)
+  parts = `git remote get-url #{name} 2>/dev/null`.match(/(\w+\/[\w-]+)(?:\.git)?$/)
```

Comment:
What happens when this runs on Windows?

Reply:
A better solution would be to use [rugged](https://github.com/libgit2/rugged), but I didn't know how y'all would feel about another gem dependency. Happy to add that if you'd prefer.

Reply:
🤔 I'm not entirely sure, happy to ask @puppetlabs/modules for input - we do have a habit of releasing tools that blow up for windows folx I'd like to avoid, if we can.

Reply:
Added it, but in a second commit so it'll be easy to choose an implementation or revert at a later date.